Gemfile.lock

```diff
@@ -98,15 +98,15 @@ GEM
       activerecord (>= 4.2, < 7)
     domain_name (0.5.20190701)
       unf (>= 0.0.5, < 1.0.0)
-    erubi (1.9.0)
+    erubi (1.10.0)
```

Comment:
### [erubi](https://github.com/jeremyevans/erubi)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.9.0 → 1.10.0

[_[change-log](https://github.com/jeremyevans/erubi/blob/master/CHANGELOG), [source-code](https://github.com/jeremyevans/erubi)_]





<details>
<summary>Commits</summary>

A change of **22** commits. See the full changes on [the compare page](https://github.com/jeremyevans/erubi/compare/1.9.0...1.10.0).

These are the first 10 commits:
- (838bc70) [Fix documentation of options bufval, bufvar in Erubi::Engine…](https://github.com/jeremyevans/erubi/commit/838bc707d41e0f01651cdd1f3e4234802382b0a3)
- (d152b2d) [Reduce memory usage when escaping text](https://github.com/jeremyevans/erubi/commit/d152b2dec928e1050d8c722a207b172bf118f4fb)
- (4dc81c2) [Refactor and simplify internals](https://github.com/jeremyevans/erubi/commit/4dc81c210664bfa244c6015bb3aa034b29f5a66f)
- (970b384) [Allow the literal prefix/postfix to be configured (Fixes #26…](https://github.com/jeremyevans/erubi/commit/970b38420b0007ccd84a0debf30da8c3e79b6252)
- (e7288b3) [Update CHANGELOG](https://github.com/jeremyevans/erubi/commit/e7288b31f337f7c00e67ac456d770fcfeab43180)
- (46a964f) [Improve rdoc formatting](https://github.com/jeremyevans/erubi/commit/46a964f7ce364c0643fa83281f818635a2716ba2)
- (5910021) [Make spec_w task use warning gem instead of egrep for filter…](https://github.com/jeremyevans/erubi/commit/5910021c09113c9249bd5d69284bc38660d79139)
- (c7a24c7) [Add nocov markers](https://github.com/jeremyevans/erubi/commit/c7a24c7ca4e99f5d6578355263e570ad3efef257)
- (3e05722) [Update copyright year](https://github.com/jeremyevans/erubi/commit/3e0572225f4d8f1663e45c238ef8e84462fc5144)
- (569da61) [Start testing Ruby 2.7 on Travis](https://github.com/jeremyevans/erubi/commit/569da6123b03699466682c76b296a09e804bd775)


</details>